crypto/fipsmodule/ec/make_tables.go

```diff
@@ -414,9 +514,40 @@ func writeU32Mont(w io.Writer, curve elliptic.Curve, n *big.Int, indent int) err
 	})
 }
 
-type writeBigIntFunc func(w io.Writer, curve elliptic.Curve, n *big.Int, indent int) error
+func writeU64(w io.Writer, curve elliptic.Curve, n *big.Int, indent int, bitSizes []uint) error {
```

Comment:
I think `bitSizes` is not used in this function nor the following one, nor in writeU32. Is there a reason to have it in the arguments?

Reply:
don't get me started on Go's limitations :) Because of the way all these functions are defined and used I couldn't find a more elegant way than adding the parameter to all the functions even if it's not used everywhere